src/timezone-handling-changes.md

```diff
@@ -4,16 +4,16 @@ Anki 2.1.22+ and AnkiMobile 2.0.57+ contain an optional change to the way Anki c
 
```

Comment:
How about the previous paragraph is changed to read the following?

"Anki 2.1.22+, AnkiMobile 2.0.57+, and AnkiDroid 2.15+ contain"

Reply:
Yes, easier :)